src/query-builder/case-builder.ts

```diff
@@ -212,6 +212,26 @@ export class CaseWhenBuilder<DB, TB extends keyof DB, W, O>
     })
   }
 
+  /**
+   * Adds an `else` clause to the `case` statement where the value is a reference to a column.
+   *
```

Comment:
let's add a link between variants.

double check it links correctly (i'm in the browser).

```suggestion
   *
   * See {@link else} for value-first variant.
```

Reply:
<img width="701" height="245" alt="image" src="https://github.com/user-attachments/assets/5366f1f4-4be4-4a22-8ae5-1bd60ef4caef" />
yep it works